apps/cli/src/adapters/node/__tests__/package-json.test.ts

```diff
@@ -1,102 +1,186 @@
 import { readFile } from "node:fs/promises";
 import { beforeEach, describe, expect, it, vi } from "vitest";
 
+import { dependenciesArraySchema, scriptsArraySchema } from "../codec.js";
 import { makePackageJsonReader } from "../package-json.js";
+import { makeMockPackageJson } from "./data.js";
 
 vi.mock("node:fs/promises", () => ({
   readFile: vi.fn(),
 }));
 
+const mockReadFile = (content: string) =>
+  (readFile as unknown as ReturnType<typeof vi.fn>).mockResolvedValueOnce(
+    content,
+  );
```

Comment:
maybe this can be simplified using `spyOn`

```js
import fs from "node:fs/promises";
// ....
const mockReadFile = vi.spyOn(fs, "readFile").mockResolvedValueOnce(content);
```

Reply:
Thanks! Changed in https://github.com/pagopa/dx/pull/691/commits/23aa318db6b48dfeb768aaf7bb1a515f9e75a5ba